core/src/main/java/org/apache/carbondata/core/datamap/DataMapProvider.java

```diff
@@ -120,7 +120,7 @@ public void initData() { }
   /**
    * Rebuild the datamap by loading all existing data from mainTable
    * This is called when refreshing the datamap when
-   * 1. after datamap creation and if `autoRefreshDataMap` is set to true
+   * 1. after datamap creation and no "WITH DEFERRED REBUILD" defined
```

Comment:
Please modify the MD document also
https://github.com/apache/carbondata/blob/master/docs/datamap/datamap-management.md#automatic-refresh

Reply:
The document does not mention `autoRefreshDataMap` now. As for the section title `Automatic Refresh`, it is explained  with `WITH DEFERRED REBUILD` in its parent section `DataMap Management`.


https://github.com/apache/carbondata/blob/2db8bf234b4234a9cbf3245ac39ed7391a84f77e/docs/datamap/datamap-management.md#L56-L64